lib/src/bubbles.dart

```diff
@@ -1,100 +1,72 @@
 import 'dart:async';
 import 'dart:math';
 
+import 'package:floating_bubbles/src/bubble_painter.dart';
+import 'package:floating_bubbles/src/bubble_animation.dart';
 import 'package:flutter/material.dart';
 import 'package:simple_animations/simple_animations.dart';
 
-import 'bubble_floating_animation.dart';
-
-///Enum for Setting the Shape of the bubble
 enum BubbleShape { circle, square, roundedRectangle }
 
-/// Creates Floating Bubbles in the Foreground of Any [widgets].
+/// Creates floating bubbles for [duration] amount of time.
+///
+/// If you want the floating bubble animation to repeat, then use the constructor
+/// `FloatingBubbles.alwaysRepeating()`.
+///
+/// [strokeWidth] is effective only if [paintingStyle] is set to [PaintingStyle.stroke].
 // ignore: must_be_immutable
 class FloatingBubbles extends StatefulWidget {
```

Comment:
Please add documentation for all of the fields.

Reply:
Done.